src/utils/validations/login.js

```diff
@@ -30,8 +30,8 @@ export const lastName = (value) => {
 }
 
 export const confirmPassword = (password, data) => {
-  let helperText
-  if (password !== data.confirmPassword) {
+  let helperText = ''
+  if (password !== data.password) {
     helperText = 'common.errorMessages.passwordsDontMatch'
   }
   helperText = emptyField(password, helperText)
```

Comment:
remove this assigning , just return emptyField(password, helperText)

Reply:
fixed